File: Ohtap E Commerce Domain Models/Development/NetFramework/Models Objects/Models Objects/DiscountDetails/DiscountDetail.cs
using Ohtap.ECommerce.Domain.Models.Objects.Languages;
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;
using System.Threading.Tasks;

namespace Ohtap.ECommerce.Domain.Models.Objects.DiscountDetails
{
    public class DiscountDetail : DiscountDetailBasic, IDiscountDetail
    {
         ILanguage IDiscountDetail.Language { get; set; }
    }
}

File: Ohtap E Commerce Domain Models/Development/PHP/Objects/DiscountDetails/DiscountDetail.php
<?php
namespace Ohtap\ECommerce\DomainModels\Objects\DiscountDetail;
use Ohtap\ECommerce\DomainModels\Objects\LanguageDetails\ILanguageDetail;
use   Ohtap\ECommerce\DomainModels\Objects\Languages;
class DiscountDetail  implements IDiscountDetail
{

   // public Language $Language;//Language sınıfını yüklemiyor bak
    public ILanguageDetail $Language;
    public string $ID;
    public string $Name;
    public int $State;
    public int $OrderNo;
    public string $Description;
}